src/parquet/file/metadata.cc

```diff
@@ -323,9 +323,9 @@ class FileMetaData::FileMetaDataImpl {
   FileMetaData::Version writer_version_;
 };
 
-std::unique_ptr<FileMetaData> FileMetaData::Make(
+std::shared_ptr<FileMetaData> FileMetaData::Make(
```

Comment:
This function is now just an alias for `make_shared`. Do we really want to keep it?